c2corg_ui/templates/helpers/view.html

```diff
@@ -401,20 +401,28 @@ <h3 class="heading closed show-phone" ng-click="mainCtrl.animateHeaderIcon($even
 
 
 ## Float buttons
-<%def name="show_float_buttons(outingUsers, doctype, doc_id, doc_lang, other_langs, missing_langs)">\
+<%def name="show_float_buttons(associatedUsers, doctype, doc_id, doc_lang, other_langs, missing_langs, image = 'null')">\
```

Comment:
Ahh, i was returning `Null` and it gave an error `"Null is not defined`" and i was like....DUUH it is not defined xd